concent_api/gatekeeper/tests/test_gatekeeper_views.py

```diff
@@ -351,6 +351,42 @@ def test_upload_should_return_401_if_specific_file_info_data(self):
         self.assertIn('message', response.json().keys())
         self.assertEqual("application/json", response["Content-Type"])
 
+    @freeze_time("2018-12-30 11:00:00")
+    def test_upload_should_return_401_if_file_category_is_not_unique(self):
```

Comment:
hm... maybe better repfrase it to sth like `...if_flie_category_is_the_same_for_both_files` or `..._not_unique_among_two_files`

even better ;)
`_not_unique_across_file_info_list`

Reply:
:+1: done